File: week6/Week6_day3_ISSS_J/app/src/main/java/com/example/luisenriquez/week6_day3_isss_j/data/RepositoryModule/Repository.java
package com.example.luisenriquez.week6_day3_isss_j.data.RepositoryModule;

import android.arch.lifecycle.MutableLiveData;

import com.example.luisenriquez.week6_day3_isss_j.data.remote.models.ISSResponse;

public interface Repository
{
    MutableLiveData<ISSResponse> ISS_RESPONSE_MUTABLE_LIVE_DATA = null;

    void getISSResponse();
}

File: Week7/Week7Day4Test/app/src/main/java/com/example/luisenriquez/week7day4test/data/repositorymodule/Repository.kt
package com.example.luisenriquez.week7day4test.data.repositorymodule

import android.arch.lifecycle.MutableLiveData
import com.example.luisenriquez.week7day4test.data.remote.model.NumbersResponse

interface Repository
{
    val numberResponseLiveData : MutableLiveData<NumbersResponse>

    fun getRandomNumbers()
}